afanasy/src/watch/itemjobtask.cpp

```diff
@@ -292,6 +372,11 @@ void ItemJobTask::taskFilesReceived( const af::MCTaskUp & i_taskup )
 		m_list->itemsHeightChanged();
 }
 
+void ItemJobTask::getTaskInfo(const std::string &i_mode, int i_number)
+{
+	ListTasks::getTaskInfo(m_block->job_id, m_blocknum, m_tasknum, i_mode, i_number);
+}
```

Comment:
Ok here ideally the content of `ListTasks::getTaskInfo()` will come, and the latter should not be called any more, but I feared dependency loops at first. Maybe solved by next commits.